### PR TITLE
Enable browser back button when full screen mode 

### PR DESCRIPTION
**Pull request by takedakun**
Fixed issue #292 and enabled browser back button when full screen mode.

### CODE REVIEW

File: src/components/renderer/renderer.tsx [hunk 1]
```diff
@@ -110,6 +110,17 @@ class Editor extends React.Component<Props, State> {
       .initialize(chart)
       .runAsync();
   }
+
+  public handlePopState(event) {
+    const pathname = Editor.pathname;
+    const urlarray = pathname.split('/');
+    const lastword = urlarray[urlarray.length - 1];
+    event.preventDefault();
+    if (this.state.fullscreen && lastword === 'view') {
```
Comment (domoritz) [inline src/components/renderer/renderer.tsx]: You can just check whether the string ends in "view"

**Comment by takedakun** [comment]
Do you mean we should push history when we push browser back button?

**Comment by domoritz** [comment]
No, I was just wondering where we push? I guess when the user opens the fullscreen modal. 

Can you update the code to check what then url ends with?

**Comment by domoritz** [comment]
No, I was just wondering where we push? I guess when the user opens the fullscreen modal. 

Can you update the code to check what then url ends with?

**Comment by takedakun** [comment]
I have one question. I think we can't use browser back button to get out of full screen modal when we in "/" or "/edited" url, because it only changes react state (this.state.fullscreen) and doesn't change browser history when opening full screen modal in those url. What do you think about this?

**Comment by domoritz** [comment]
That's why I asked about `pushState`. Anyway, it sounds like this needs some more work and we probably want a solution that works for all routes and not just fullscreen so I'm going to close this. Please open a new PR.

**Pull request closed without merging**